packages/core-modules/contracts/modules/AnotherModuleMock.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../mixins/CoreCommsMixin.sol";
+import "./SomeModuleMock.sol";
+
+contract AnotherModuleMock is CoreCommsMixin {
```

Comment:
To commit to the rule of not having mocks in production code areas, could we rename `AnotherModuleMock.sol` and `SomeModuleMock.sol` to `SampleModuleA.sol` and `SampleModuleB.sol`?

They'd still be mocks in a way, but the naming makes it more acceptable IMO. My brain simply rejects anything with the word "mock" in production folders.